src/components/header-bar/header-bar.jsx

```diff
@@ -59,13 +62,38 @@ export const HeaderBar = ({
                 window.location.href = getPath(app.defaultAction)
             },
         }))
-    }, [data, baseUrl])
+    }, [data, getPath])
 
     // fetch commands
     const commands = []
 
     // fetch shortcuts
-    const shortcuts = []
+    const shortcuts = useMemo(() => {
+        if (!data?.apps?.modules) {
+            return []
+        }
+
+        return data.apps.modules?.reduce((acc, currModule) => {
+            const { defaultAction, icon } = currModule
+            const shortcuts =
+                currModule.shortcuts?.map(({ name, url }) => {
+                    const shortcutDefaultAction = getPath(defaultAction) + url
+                    return {
+                        type: SHORTCUT,
+                        name,
+                        // ToDo: confirm what the default action should be in Global shell
```

Comment:
This works for now since it will redirect.  We'll probably want to update it later to use a client-side route.